docs/guide/deployment/certificate-authority.md

```diff
@@ -0,0 +1,129 @@
+# Certificate authority
+This service is used to sign certificates for devices and [plgd-dev/sdk](https://github.com/plgd-dev/sdk) client.
+
+> During onboarding device via SDK (at first SDK gets OCF identity certificate for self), SDK ask the service to sign CSR (Certificate Signed Request) for the device and when it is success, the service returns certificate chain.
```

Comment:
Initialization of the SDK requires an Identity Certificate, issued by this service after successful user authorization. The SDK is then able to discover and configure the ownership as described [here](https://openconnectivity.org/specs/OCF_Security_Specification_v2.2.1.pdf#page=38). During this process, a CSR (Certificate Signing Request) is sent to the Certificate Authority which returns device's Identity Certificate.

e.g. plgd mobile application uses SDK for device ownership configuration. Available on both [Google Play](https://play.google.com/store/apps/details?id=dev.plgd.client&hl=sk&gl=US) and [Apple Store](https://apps.apple.com/sk/app/plgd/id1536315811).